src/librustc_target/spec/armv7_none_eabihf.rs

```diff
@@ -0,0 +1,35 @@
+// Generic arm target for bare-metal code
+//
+// Can be used in conjunction with the `target-feature` and
+// `target-cpu` compiler flags to opt-in more hardware-specific
+// features.
+//
+// For example, `-C target-cpu=cortex-a7`.
+
+use crate::spec::{LinkerFlavor, LldFlavor, PanicStrategy, Target, TargetOptions, TargetResult};
+
```

Comment:
You probably want `+strict-align` for bare-metal.

It can bite you with a hard to find bugs if alignment errors happen. Made the experience already: https://github.com/rust-lang/rust/commit/c4e46140f6a295d2d668ccc5bf25858233b11cbb